repository/a.json

```diff
@@ -2306,6 +2306,18 @@
 				}
 			]
 		},
+		{
+			"name": "AutoPEP8_2020",
```

Comment:
What is the 2020 for here? Also spaces are allowed in names. So why not "AutoPEP8"?

Reply:
Because there is already a plug-in called `AutoPEP8` in the repository, and this `AutoPEP8` has stopped updating, I just updated it.